api/v1/relay/relay.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package relay;
+
+// NodeStatusEvent is a message sent by hubble-relay to inform clients about
+// the state of a particular node.
+message NodeStatusEvent {
+    // state_change contains the new node state
+    NodeState state_change = 1;
+    // node_name is the list of nodes for which the above state changes applies
+    repeated string node_name = 2;
+}
+
+enum NodeState {
+    // UNKNOWN_NODE_STATE indicates that the state of this node is unknown.
+    UNKNOWN_NODE_STATE = 0;
```

Comment:
UNKNOWN needs to be prefixed, because protobuf does not have enum namespaces and will therefore collide with any other enum which will have the UNKNOWN variant. But I guess for the others, we can do that.

Reply:
I have shortened the variants to `NODE_xxx`. I think a prefix is still useful, since `relay.proto` will gain quite a few other enums that maybe also want to make use of variants like `ERROR`.